README.md

```diff
@@ -1,81 +1,69 @@
 # nvim-lspconfig
 
-Collection of common configurations for the [Nvim LSP client](https://neovim.io/doc/user/lsp.html).
-
-## Best-effort, unsupported
-
+A collection of common configurations for the [Nvim LSP client](https://neovim.io/doc/user/lsp.html).
 The configurations here are strictly **best effort and unsupported**.
 
-This repo is (1) a place for Nvim LSP users to collaboratively provide starting
-examples for the many LSP backends out there, and (2) a reference for the
-current best practice (or most popular) regarding choice of server, setup, etc.
-
-## Requires Nvim 0.5 HEAD
-
-While Nvim LSP undergoes development, the configs in this repo assume that you
-are using the latest [Nvim HEAD/nightly build](https://github.com/neovim/neovim/releases/tag/nightly).
+This repo is :
 
-Update Nvim and nvim-lspconfig before reporting an issue.
+* a place for Nvim LSP users to collaboratively provide starting 
```

Comment:
I think `a place for Nvim LSP client users` sounds better, or maybe `a place for Nvim builtin LSP client users` ?

Reply:
This is always something we've been unclear about. We should make it clear that neovim implements a language server client, in accordance with language server protocol, via which it communicates with language servers.